domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -238,7 +248,7 @@ private static async Task InsertTransferAgreementHistoryEntry(ApplicationDbConte
         var historyQuery =
             $"INSERT INTO \"{historyTable}\" (\"Id\", \"CreatedAt\", \"AuditAction\", \"ActorId\", \"ActorName\", \"TransferAgreementId\", \"StartDate\", \"EndDate\", \"SenderId\", \"SenderName\", \"SenderTin\", \"ReceiverTin\") " +
             "VALUES (@Id, @CreatedAt, @AuditAction, @ActorId, @ActorName, @TransferAgreementId, @StartDate, @EndDate, @SenderId, @SenderName, @SenderTin, @ReceiverTin)";
-        var historyFields = new[]
+        object[] historyFields =
```

Comment:
And this?

Reply:
Dotnet 8, introduces new warnings, that are resolved, by doing it like this.